features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines application features from the specific context.
+ */
+class FeatureContext extends RawDrupalContext implements SnippetAcceptingContext {
+
+  protected $client;
+
+  protected $verb;
+
+  protected $endpoint;
+
+  protected $headers = [];
+
+  protected $request;
+
+  protected $response;
+
+  /**
+   * Constructor.
+   */
+  public function __construct() {
+    $this->client = new \GuzzleHttp\Client(['base_uri' => 'http://127.0.0.1:8080/']);
+  }
+
+  /**
```

Comment:
as discussed yesterday, these steps should be ported to plain PHPUnit (but it's still a good idea to separate them from the test classes to make the code reusable)